app/views/grouped_issue_mailer/error_occurred.html.haml

```diff
@@ -1,6 +1,6 @@
 .col-lg-6
   %p.gray_text{:style => 'display:inline'} Project:
-  %small #{@issue.website.title}
+  %small [#{@issue.website.title}] #{truncate(@issue.message, length: 40)}
```

Comment:
You seem to have no understanding of what i mean by subject. This string should be in the mailer for the subject attribute. That's where this shows up in the email For the email subject!! ...